proto/osmosis/authenticator/params.proto

```diff
@@ -7,8 +7,20 @@ option go_package = "github.com/osmosis-labs/osmosis/v23/x/authenticator/types";
 
 // Params defines the parameters for the module.
 message Params {
+  // MaximumUnauthenticatedGas defines the maximum amount of gas that can be
+  // used to authenticate a transaction in ante handler without having fee payer
+  // authenticated.
   uint64 maximum_unauthenticated_gas = 1
       [ (gogoproto.moretags) = "yaml:\"maximum_unauthenticated_gas\"" ];
-  bool are_smart_accounts_active = 2
-      [ (gogoproto.moretags) = "yaml:\"cosigner_contract\"" ];
+
+  // AuthenticatorActiveState defines the state of the authenticator.
+  // If set to false, the authenticator module will not be used
+  // and the classic cosmos sdk authentication will be used instead.
+  bool authenticator_active_state = 2
```

Comment:
I did like the smart_accounts_active name from before better

Reply:
I don't mind either way, only that we should have consistent naming convention referring to the same thing. (`authenticator` vs `smart_account`)

Reply:
I think we are going to rename the module to `smart-account` soon, so would be good to keep the name! 

Should have mentioned this eariler, will add a task to our task list now!

Reply:
Oh cool! I will change the name back then!